pymc/distributions/transforms.py

```diff
@@ -312,7 +312,7 @@ def extend_axis(array, axis):
     def extend_axis_rev(array, axis):
         normalized_axis = normalize_axis_tuple(axis, array.ndim)[0]
 
-        n = array.shape[normalized_axis].astype("floatX")
+        n = pt.cast(array.shape[normalized_axis], "floatX")
```

Comment:
Instead you can call `pt.as_tensor(array)`

Reply:
I'm wondering, is it better than pt.cast for some reason?

Reply:
Not really, I was just thinking out loud